buildenv/jenkins/openjdk_tests

```diff
@@ -104,6 +104,10 @@ def PLATFORM_MAP = [
     'x86-64_windows_xl' : [
         'SPEC' : 'win_x86-64',
         'LABEL' : 'ci.role.test&&hw.arch.x86&&sw.os.windows',
+    ],
+    'riscv64_linux' : [
+        'SPEC' : 'linux_riscv-64_cmprssptrs',
```

Comment:
This should match the spec listed in openj9/buildspecs, so https://github.com/eclipse/openj9/blob/master/buildspecs/linux_riscv64_cmprssptrs.spec (no '-' in the name).